src/ViewHelpers/ReplaceViewHelper.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+/**
+ * The ReplaceViewHelper replaces one or multiple strings with other
+ * strings. This ViewHelper mimicks PHP's :php:`str_replace()` function.
+ * However, it's also possible to provide replace pairs as associative array
+ * via the "replace" argument.
+ *
+ *
+ * Examples
+ * ========
+ *
+ * Replace a single string
+ * -----------------------
+ * ::
+ *
+ *    <f:replace value="Hello World" search="World" replace="Fluid" />
+ *
+ * .. code-block:: text
+ *
+ *    Hello Fluid
+ *
+ *
+ * Replace multiple strings
+ * ------------------------
+ * ::
+ *
+ *    <f:replace value="Hello World" search="{0: 'World', 1: 'Hello'}" replace="{0: 'Fluid', 1: 'Hi'}" />
+ *
+ * .. code-block:: text
+ *
+ *    Hi Fluid
+ *
+ *
+ * Replace multiple strings using associative array
+ * ------------------------------------------------
+ * ::
+ *
+ *    <f:replace value="Hello World" replace="{'World': 'Fluid', 'Hello': 'Hi'}" />
+ *
+ * .. code-block:: text
+ *
+ *    Hi Fluid
+ */
+final class ReplaceViewHelper extends AbstractViewHelper
+{
+    use CompileWithRenderStatic;
+
+    public function initializeArguments(): void
+    {
+        $this->registerArgument('value', 'string', '', false);
+        $this->registerArgument('search', 'mixed', '', false);
+        $this->registerArgument('replace', 'mixed', '', true);
+    }
+
+    public static function renderStatic(array $arguments, \Closure $renderChildrenClosure, RenderingContextInterface $renderingContext)
+    {
+        $value = $arguments['value'] ?? $renderChildrenClosure();
+        $search = $arguments['search'];
+        $replace = $arguments['replace'];
+
+        if ($value === null) {
```

Comment:
we should probably check if $value can be converted to a string here, as well.
this however is an issue in other VH's as well, and we may want to take care of this in a more global fashion.

something like:
function can_be_string($var) {
    return $var === null || is_scalar($var) || $var instanceof Stringable);
}